incubator/demo-ycsb/README.md

```diff
@@ -146,7 +146,7 @@ replicationcontroller "demo-ycsb" scaled
 To uninstall/delete the deployment:
 
 ```bash
-helm del --purge ycsb
+helm del ycsb
```

Comment:
Maybe better t use the full command version `helm delete ycsb`?

Reply:
Yes, I think so too, thanks for your suggestion. I also changed in all other locations. "del" was used in all the chart clean-up sections.